compiler-core/src/error.rs

```diff
@@ -2258,16 +2258,16 @@ package to Hex.\n"
                         .to_string();
                 text.push_str(if *description_missing && *licence_missing {
                     r#"Add the licences and description fields to your gleam.toml file.
-                
+
```

Comment:
These changes are due to my IDE settings, I can revert them if needed.

Reply:
nah this is an improvement 👍